src/Camera/QGCCameraControl.cc

```diff
@@ -1967,6 +1969,42 @@ QGCCameraControl::_handleDefinitionFile(const QString &url)
     emit dataReady(bytes);
 }
 
+//-----------------------------------------------------------------------------
+void
+QGCCameraControl::_handleHeartbeat (mavlink_message_t& message)
+{
+    // Only perform the following steps for Anafi heartbeats
+    if(message.sysid != 1 || message.compid != 41)  {
+        return;
+    }
+
+    uint32_t custom_mode = mavlink_msg_heartbeat_get_custom_mode(&message);
+    switch(custom_mode) {
```

Comment:
Not a big beal, but might be nice to use the same `enum` as here: 
https://github.com/planckaero/planck_ctrl/blob/master/include/anafi/AnafiSubmode.h#L60-L67
(and add the link in comment)